vendor/github.com/prometheus/prom2json/Dockerfile

```diff
@@ -1,2 +1,3 @@
-FROM        sdurrheimer/alpine-golang-make-onbuild
-MAINTAINER  The Prometheus Authors <prometheus-developers@googlegroups.com>
+# No other revision than latest is available
+# hadolint ignore=DL3007
+FROM        sdurrheimer/alpine-golang-make-onbuild:latest
```

Comment:
I don't think we can change vendor, we probably just want to ignore it in hadolint

Reply:
Docker "MAINTAINER" instruction is deprecated

Reply:
Its in the vendor directory, we cannot change it. It will be overridden the next time someone runs `go mod vendor`

Reply:
ohhhhh right, I totally missed that.

Reply:
I'll exclude vendors from linting and remove the 2 vendor changed files

Reply:
I'll remove vendors from linting and revert my changes on those